### PR TITLE
Load defaults from a bundle

### PR DESCRIPTION
**Pull request by jzelenjak**
# Description of changes made
- Create a `.properties` file with default values for quick access parameters
- Create `TestGenieDefaultsBundle` to load these default values
- Update `QuickAccessParametersState` class to use the bundle
- Change some references to *ToolWindowState* to *QuickAccessParametersState*

# Why is merge request needed
This merge request is needed to make our plugin more maintainable: instead of hardcoding the default values, they are loaded from a file. In case the default values change in EvoSuite, it will be much easier to update them in our code.

# Other notes
Closes #84 

# What is missing?

- [x] I have checked that I am merging into correct branch

### CODE REVIEW

File: src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/toolwindow/QuickAccessParameters.kt [hunk 1]
```diff
@@ -59,7 +59,7 @@ class QuickAccessParameters(_project: Project) {
     private var stoppingConditionToolTip =
         JBLabel("Default: 60 seconds", UIUtil.ComponentStyle.SMALL, UIUtil.FontColor.BRIGHTER)
     private var populationLimitToolTip =
-        JBLabel("Default: 60 seconds", UIUtil.ComponentStyle.SMALL, UIUtil.FontColor.BRIGHTER)
+        JBLabel("Default: 50 seconds", UIUtil.ComponentStyle.SMALL, UIUtil.FontColor.BRIGHTER)
```
Comment (martinmladenov) [inline src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/toolwindow/QuickAccessParameters.kt:62]: Why did you change this?

**Review by martinmladenov — APPROVED** [review]
It seems very nice, thanks for taking the time to implement it. It will certainly make managing default parameters much easier :)

File: src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/toolwindow/QuickAccessParameters.kt (hunk 1 above)
Comment (jzelenjak) [reply src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/toolwindow/QuickAccessParameters.kt:62]: https://github.com/EvoSuite/evosuite/blob/master/client/src/main/java/org/evosuite/Properties.java

**Review by kirilvasilev16 — APPROVED** [review]
Good job!

File: src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/toolwindow/QuickAccessParameters.kt (hunk 1 above)
Comment (martinmladenov) [reply src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/toolwindow/QuickAccessParameters.kt:62]: Ah, I see. Thanks for clearing this up.
